package.json

```diff
@@ -205,20 +205,21 @@
 		"publish:dev": "npm run build:packages && lerna publish --npm-tag next",
 		"publish:prod": "npm run build:packages && lerna publish",
 		"test": "npm run lint && npm run test-unit",
-		"pretest-e2e": "cross-env SCRIPT_DEBUG=false ./bin/reset-local-e2e-tests.sh",
```

Comment:
What happens to the database after all e2e tests finish their execution?

This command was used to reset all created posts, taxonomies, etc so they don't have an impact on the result of subsequent runs of the same tests.

Reply:
Thanks for pointing that one out, @gziolo! 009f02c72 implements an equivalent reset before running e2e tests.